community-operators/postgresql/4.1.0/postgresoperator.v4.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,549 @@
+        - urn:alm:descriptor:com.tectonic.ui:podCount
+      statusDescriptors:
+      - description: The current status of the application.
+        displayName: Status
+        path: phase
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.phase
+      - description: Explanation for the current status of the application.
+        displayName: Status Details
+        path: reason
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.phase:reason
+      version: v1
+    - description: Represents a Postgres sql policy
+      displayName: Postgres SQL Policy
+      kind: Pgpolicy
+      name: pgpolicies.crunchydata.com
+      resources:
+      - kind: Pgpolicy
+        name: ''
+        version: v1
+      specDescriptors:
+      - description: The pgpolicy name.
+        displayName: Name
+        path: name
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:name
+      - description: The pgpolicy sql.
+        displayName: SQL
+        path: sql
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:sql
+      statusDescriptors:
+      - description: The current status of the application.
+        displayName: Status
+        path: phase
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.phase
+      - description: Explanation for the current status of the application.
+        displayName: Status Details
+        path: reason
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.phase:reason
+      version: v1
+    - description: Represents a Postgres workflow task
+      displayName: Postgres workflow task
+      kind: Pgtask
+      name: pgtasks.crunchydata.com
+      resources:
+      - kind: Pgtask
+        name: ''
+        version: v1
+      specDescriptors:
+      - description: The pgtask name.
+        displayName: Name
+        path: name
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:name
+      - description: The pgtask type.
+        displayName: TaskType
+        path: tasktype
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:tasktype
+      statusDescriptors:
+      - description: The current status of the application.
+        displayName: Status
+        path: phase
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.phase
+      - description: Explanation for the current status of the application.
+        displayName: Status Details
+        path: reason
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.phase:reason
+      version: v1
+    - description: Represents a Postgres backup task
+      displayName: Postgres backup task
+      kind: Pgbackup
+      name: pgbackups.crunchydata.com
+      resources:
+      - kind: Pgbackup
+        name: ''
+        version: v1
+      specDescriptors:
+      - description: The pgbackup name.
+        displayName: Name
+        path: name
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:name
```

Comment:
Similarly, recommend using `urn:alm:descriptor:com.tectonic.ui:text` (see [details in "Text Input"](https://github.com/openshift/console/blob/c07050bccf2e2c1d94001d138a063f6d4206043b/frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md#text-input)) as the `x-descriptors`.